minorminer/layout/placement.py

```diff
@@ -0,0 +1,60 @@
+import networkx as nx
+from scipy.spatial.distance import euclidean
+
+
+def closest(S_layout, T_layout):
+    """
+    Maps vertices of S to the closest vertices of T as given by S_layout and T_layout. i.e. For each vertex u in 
+    S_layout and each vertex v in T_layout, map u to the v with minimum Euclidean distance (||u - v||_2).
+
+    Parameters
+    ----------
+    S_layout : dict
+        A layout for S; i.e. a map from S to the plane.
+    T_layout : dict
+        A layout for T; i.e. a map from T to the plane..
+
+    Returns
+    -------
+    placement : dict
+        A mapping from vertices of S (keys) to vertices of T (values).
+    """
+    placement = {}
+    for u, u_pos in S_layout.items():
+        placement[u] = min(
+            T_layout, key=lambda v: euclidean(u_pos, T_layout[v]))
+    return placement
+
+
+def injective(S_layout, T_layout):
+    """
+    Injectively maps vertices of S to the closest vertices of T as given by S_layout and T_layout. i.e. This is the 
```

Comment:
"and T_layout. i.e. This is" --> either "and T_layout; i.e. this is" or else "and T_layout. This is"